cip-XXXX-Synchronizer-Migration-to-Canton-3-3/cip-XXXX-Synchronizer-Migration-to-Canton-3-3.md

```diff
@@ -0,0 +1,99 @@
+- Application developers need stable and easily accessible APIs to build on. Canton 3.3 takes a major step forward in stabilizing the developer facing APIs of the Canton participant node by making all calls available in the gRPC Ledger API also accessible via HTTP JSON with OpenAPI schema definitions being provided for easy discoverability. Thus allowing developers to freely pick between gRPC or HTTP JSON depending on what suits them best.
+- The Canton 3.3 release also introduces experimental Daml support for verifying cryptographic signatures. Thereby making it easier to build Daml workflows that bridge between Canton Network and other chains.
+
+## Rationale 
+The Canton 3.3 release is available for upgrade.  It enables new features in Splice 0.4.0, as well as new capabilities to be leveraged by Canton Network application developers and partners. A preview of the Canton 3.3 release notes is available [here](https://blog.digitalasset.com/developers/release-notes/canton-daml-3.3-preview).
+
+The Canton 3.3 release introduces the following new capabilities:
+- Easier application upgrades:  [Smart contract upgrading (SCU)](https://docs.daml.com/upgrade/smart-contract-upgrades.html#smart-contract-upgrade) that allows Daml models (packages in DAR files) to be updated transparently, provided guidelines in making application changes are followed.  
+- Support for the Canton Network Token Standard. SCU allows the CNTS to evolve without breaking changes.                                                                      
+- Enhancements for the interactive submission service (e.g., Time boundaries functions).  The interactive submission service is enabled by default.
+- The Canton JSON APv2 is available which is as fully featured as the gRPC LAPI: mirroring the LAPI’s gRPC functionality (e.g., ability to submit batches of commands, Daml interfaces). It follows industry standards such as [AsyncAPI](https://docs.digitalasset.com/build/3.3/explanations/json-api/asyncapi) (websocket) and [OpenAPI](https://docs.digitalasset.com/build/3.3/explanations/json-api/openapi.html) (synchronous, request-response), which enables the use of the associated tooling (e.g., generate language bindings at the discretion of the developer). 
+- A new approach to export and import of ACS snapshots, which is an improvement for any future [Synchronizer Upgrades with Downtime](https://docs.dev.sync.global/sv_operator/sv_major_upgrade.html#sv-upgrades), and will prepare Canton for zero-downtime protocol upgrades.
```

Comment:
I think that's a bit misleading given that splice is not yet using those endpoints.

Reply:
Changed to "A new approach to export and import of ACS snapshots that Splice can make use for  Synchronizer Upgrades with Downtime."